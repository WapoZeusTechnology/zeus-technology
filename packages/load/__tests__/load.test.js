'use strict';

const load = require('..');

describe('@zeus/load', () => {
    it('needs tests');
});
