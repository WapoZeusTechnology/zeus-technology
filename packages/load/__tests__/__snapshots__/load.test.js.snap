// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@zeus-platform/load Should render with KVPs and match snapshot 1`] = `"<scipt crossorigin=\\"true\\" src=\\"https://foo.com\\" async=\\"true\\" id=\\"__zeus_script\\"></script><script type=\\"application/javascript\\">document.getElementById(\\"__zeus_script\\").onload=function(){window.zeus.run()}"`;
